packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.tsx

```diff
@@ -47,18 +47,22 @@ export class FocusTrapZone extends React.Component<IFocusTrapZoneProps, {}> impl
   public componentDidUpdate(prevProps: IFocusTrapZoneProps) {
     const prevForceFocusInsideTrap = prevProps.forceFocusInsideTrap !== undefined ? prevProps.forceFocusInsideTrap : true;
     const newForceFocusInsideTrap = this.props.forceFocusInsideTrap !== undefined ? this.props.forceFocusInsideTrap : true;
+    const prevDisabled = prevProps.disabled !== undefined ? prevProps.disabled : false;
```

Comment:
Suggestion, deconstruct prev props and this.props so you don't have to keep doing this. and prevProps.

Reply:
It's really only done on this line and consistent with other code. Generally if it's just one line, I don't deconstruct (since it's net less lines of code.)

Reply:
(Also, I think `this` and `prevProps` context is important here.)